src/RestRouteList.php

```diff
@@ -0,0 +1,67 @@
+<?php declare (strict_types = 1);
+
+namespace OdbavTo\PresenterRoute;
+
+use Nette;
+use Nette\Application\Request;
+use Nette\Http\IRequest;
+
+class RestRouteList extends Nette\Application\Routers\RouteList
+{
+	const ALLOWED_METHODS_KEY = 'methods';
```

Comment:
It would be nice to use something more describing instead of `methods`, etc `allowed_http_methods`